### ISSUE TITLE
Bug parse delta to html

### ISSUE THREAD

**Issue by sald19** [deanblackborough/php-quill-renderer#30]
Hi Dean, i found this bug parsing delta with only one `\n`

![](https://i.imgur.com/8pd0EVA.png)

the text generate this JSON Delta
```javascript
{
	"ops": [{
		"insert": "normal text here\nbullet 1"
	}, {
		"attributes": {
			"list": "bullet"
		},
		"insert": "\n"
	}, {
		"insert": "bullet 2"
	}, {
		"attributes": {
			"list": "bullet"
		},
		"insert": "\n"
	}, {
		"insert": "bullet3"
	}, {
		"attributes": {
			"list": "bullet"
		},
		"insert": "\n"
	}]
}
```

render HTML
![](https://i.imgur.com/aja8v2j.png)

```html
<ul>
    <li>normal text herebullet 1</li>
    <li>bullet 2</li>
    <li>bullet3</li>
</ul>
```

**Comment by deanblackborough** [comment]
Thank you Carlos.

The else condition in the splitdeltas method doesn't account for that example, I will have a look over the weekend.

**Comment by sald19** [comment]
thanks

**Comment by deanblackborough** [comment]
Fixed in v1.01.1, not entirely happy with the solution, will look at in again when I refactor for v2.0.0
